CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - `#![no_std]` is now supported out of the box. (You don’t need to opt into any features, it just works.)
+- [**BREAKING**] a `default_code` expression can now refer to the values of earlier fields by name.
+  (This is extremely unlikely to break your code, but could in theory due to shadowing.)
+- `#[builder(exclude)]` on fields, to not provide a method to set that field.
+- Control of documentation:
+  - `#[builder(doc = "…")]` on fields, to document the field’s method on the builder. Unlike `#[doc]`, you can currently only have one value rather than one attribute per line; but that’s not a big deal since you don’t get to use the `///` sugar anyway. Just use a multiline string.
+  - `#[builder(doc, builder_method_doc = "…", builder_type_doc = "…", build_method_doc = "…")]` on structs:
```

Comment:
This style seems a bit cumbersome... wouldn't it be better to use nested attributes?
```rust
#[builder(builder_method_doc = "...", builder_type_doc(type_doc = "...", build_method_doc = "..."))]
```
And when you want to automatically create the docs for the builder type, just use:
```rust
#[builder(builder_method_doc = "...", builder_type_doc())]
```
or even
```rust
#[builder(builder_method_doc = "...", builder_type_doc)]
```

Reply:
Well, I had `doc` as implied by either of the other two, so it would normally either be `#[builder(doc)]` or the likes of `#[builder(builder_type_doc = "…")]`.

So many things starting with the letters “build” is bad for health!

We could do it with nested attributes, but I don’t think it’s particularly better or worse. Flat can be easier to read and remember.